src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+export const Tabs = (
+  { tabs,
+    selectedTabId,
+    onTabSelected },
+) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(el => (
+          <li
+            data-cy="Tab"
+            className={(selectedTabId === el.id) ? 'is-active' : ''}
+          >
+            <a
+              href={`#${el.id}`}
+              data-cy="TabLink"
+              onClick={(e) => {
+                if (selectedTabId !== el.id) {
+                  onTabSelected(el);
+                }
+              }}
```

Comment:
Move to separate function (transfer the function to a variable, and here only write the name of the function)